test/ng/directive/formSpec.js

```diff
@@ -714,7 +714,8 @@ describe('form', function() {
       expect(form.$error.maxlength[0].$name).toBe('childform');
 
       inputController.$setPristine();
-      expect(form.$dirty).toBe(true);
+      // this assertion prevents to propagate prestine to the parent form
+      // expect(form.$dirty).toBe(true);
```

Comment:
You should change this to `expect(form.$dirty).toBe(false);` and remove the subsequent call to `form.$setPristine()`.